contracts.md

```diff
@@ -12,20 +12,29 @@ liquidity for OHM on Etherscan.
 
 ## sOHM
 
-https://etherscan.io/address/0x31932E6e45012476ba3A3A4953cbA62AeE77Fbbe
+https://etherscan.io/address/0x04f2694c8fcee23e8fd0dfea1d4f5bb8c352111f
 
 You receive sOHM when you stake OHM at a 1:1 ratio. Adding this address to your
 wallet allows you to track your sOHM balance which increases with every rebase.
+Below are listed old sOHM contracts which got phased out due to historical
+reasons.
+
+- [0x3193...Fbbe](https://etherscan.io/address/0x31932E6e45012476ba3A3A4953cbA62AeE77Fbbe)
```

Comment:
Trying to establish a consistent list pattern since there are some contracts already having multiple outdated versions.